src/components/InfoPanel.component.js

```diff
@@ -137,6 +171,25 @@ class InfoPanel extends Component {
           </div>
         )}
 
+        {!!media.length &&
+          isEditing &&
+          isRealTimePOI && (
+            <FieldGroup
+              inputType="text"
+              value={captions}
+              placeholder="Change Media Caption Here"
+              onChange={captions =>
+                modifyPOIFormCaption(
+                  this.props.carouselIndex,
```

Comment:
since `carouselIndex` is already destructured, you dont need to use `this.props` here